### PR TITLE
Add datastore getting started docs for exposqliteadapter

### PR DESCRIPTION
**Pull request by chintannp**
_Issue #, if available:_

_Description of changes:_
This PR adds the instructions for using ExpoSQLiteAdapter for datastore for Expo CLI built apps.
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx [hunk 1]
```diff
@@ -56,5 +56,40 @@ DataStore.configure({
...
+**Use SQLite for enhanced performance (optional)**
+
+Expo built apps can now use SQLite with DataStore. SQLite offers considerable performance advantages over the default "AsyncStorage" database.
+
+To enable SQLite with DataStore for enhanced local database performance, follow the steps below:
+
+```sh
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npm install aws-amplify @aws-amplify/datastore-storage-adapter expo-sqlite aws-amplify-react-native amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
```
Comment (Samaritan1011001) [inline src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:80]: Is it essential to install these to get started?
`aws-amplify-react-native`, `amazon-cognito-identity-js` , `@react-native-picker/picker`

Comment (cshfang) [inline src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:80]: To build on @Samaritan1011001's point, why are these steps different than the ones above which makes use of `npx amplify-app@latest`? Shouldn't it basically be the same steps only with the addition of `@aws-amplify/datastore-storage-adapter expo-sqlite`?

File: src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx [hunk 2]
```diff
@@ -56,5 +56,40 @@ DataStore.configure({
...
+</Block>
+<Block name="Expo">
+
+Start with the [Expo](https://docs.expo.dev/):
+
+```bash
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npx amplify-app@latest
+npm install @react-native-community/netinfo @react-native-async-storage/async-storage
```
Comment (cshfang) [inline src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:68]: Should this be `expo install`?

File: src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx (hunk 1 above)
Comment (chintannp) [reply src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:80]: Good point, `aws-amplify-react-native` and `@react-native-picker/picker` will not be needed. However, `amazon-cognito-identity-js` is needed as aws-amplify depends on it. I'll remove those additional packages.

Comment (chintannp) [reply src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:80]: Yes, I copied the steps from existing guide and it is missing the `npx amplify-app@latest`. I'll add that step to both React Native and Expo steps.

File: src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx (hunk 2 above)
Comment (chintannp) [reply src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx:68]: Yes, we should use `expo install`. So, it uses the compatible versions with the Expo SDK.

**Review by Samaritan1011001 — APPROVED** [review]
LGTM, thank you!

**Review by cshfang — APPROVED** [review]

**Pull request merged**